vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridDetailsRowIT.java

```diff
@@ -25,7 +25,6 @@
 import org.openqa.selenium.WebElement;
 
 import java.util.List;
-import java.util.Locale;
```

Comment:
Changes to GridDetilaRow tests are a cherry-pick of https://github.com/vaadin/flow-components/pull/2611/commits/57c2b3de08df7e4e0a32a0f889dbdad8142b0172 and not directly related to this PR